stories/setup/Provider.js

```diff
@@ -14,7 +15,7 @@ const history = addQueryParamsToHistory({
 
 export default function Provider({ story }: Object) {
   return (
-    <Root store={store} history={history} i18n={fakeI18n}>
+    <Root store={store} history={history} i18n={fakeI18n} _config={config}>
```

Comment:
That's not needed. If Flow complains, then make the prop optional in `Root` using `_config?: typeof config`.